src/prefect/orion/models/deployments.py

```diff
@@ -86,6 +86,7 @@ async def create_deployment(
                         "path",
                         "entrypoint",
                         "infra_overrides",
+                        "parameter_openapi_schema"
```

Comment:
Great catch!

I think we should be explicitly excluding instead of including here, as this is likely to break in the future when we add more fields.

Could we change
```python 
set_={
                **deployment.dict(
                    shallow=True,
                    include={
...
```

to

```python
set_={
                **deployment.dict(
                    shallow=True,
                    exclude_unset=True,
                    exclude={"id", "created", "updated"}
}
```

Reply:
```"updated"``` should be included I assume? Otherwise, Last Updated will not change in the Orion Database